README.md

```diff
@@ -3,6 +3,6 @@ Prometheus exporter for machine/system/host/node-related metrics.
 
 Several people have been confused by the repository name `node_exporter`,
 thinking it was an exporter for [Node.js](http://nodejs.org/). Consequently,
-they assumed that there wasn't a machine metrics exporter yet. This is why
-this dummy repo was created, to catch people's glances and redirect them to
+they assumed that there wasn't a machine metrics exporter, yet. This is why
```

Comment:
It's not needed, but valid. I don't really feel strongly either way, though.

Richard

Sent by mobile; excuse my brevity.